app/routes/notifications-setup.routes.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const NotificationsSetupController = require('../controllers/notifications-setup.controller.js')
+
+const basePath = '/notifications/setup/'
+
+const routes = [
+  {
+    method: 'GET',
+    path: basePath + 'returns-period',
+    options: {
+      handler: NotificationsSetupController.viewReturnsPeriod,
+      auth: {
+        access: {
+          scope: ['returns']
```

Comment:
I think this is the only auth access needed for this page. There are some notification scopes that could be added ?

Reply:
I'd have to check the scopes used on the legacy pages. But this is fine whilst we scaffold things out.